app/views/books/index.html.erb

```diff
@@ -0,0 +1 @@
+<div id = "books_list" data = "<%= @books.to_json %>" />
```

Comment:
Remove spaces around `=`

Reply:
You don't need this line if you will fetch data over API (highly recommended)

Reply:
Agree. The problem is the data doesn't fetch from API yet